watertap/tools/parameter_sweep/parameter_sweep_differential.py

```diff
@@ -246,13 +247,14 @@ def _run_differential_sweep(self, model, local_value):
             reinitialize_function=self.config.reinitialize_function,
             reinitialize_kwargs=self.config.reinitialize_kwargs,
             reinitialize_before_sweep=self.config.reinitialize_before_sweep,
+            parallel_manager_class=SingleProcessParallelManager,
             comm=DummyCOMM,
```

Comment:
Is the `DummyCOMM` still needed given that SingleProcessParallelManager exists?

Reply:
It is for the moment, unfortunately - since not all explicit uses of the `comm` object are pulled out of parameter sweep. We'll be able to remove it once I get rid of the other uses.

Reply:
@shelman Do you intend to make this change in the PR or a subsequent one?

Reply:
A subsequent one!